src/peergos.css

```diff
@@ -571,7 +571,6 @@ h3 {word-wrap:break-word; white-space:normal;}
 }
 
 .flex-item > div {
-  width: 300px;
```

Comment:
does this affect anything else?

Reply:
flex-item only used on profile + search page. 
On the profile page the change means the bio & status field is sized more appropriately.  
Have confirmed style is fine on search page.
I also made a small style change on search page to make the search type dropdown more prominent.